build.gradle

```diff
@@ -100,27 +100,79 @@ model {
     }
 }
 
+task copyDeployDir {
+    String prefix = 'test-results/frcUserProgramTest/'
+    String suffix = 'src/main/deploy'
 
-tasks.register('copyDeployDirForUnitTestsDebug', Copy) {
-    from project.fileTree('src/main/deploy')
-    if(OperatingSystem.current().isWindows()){
-        into layout.buildDirectory.dir("test-results/frcUserProgramTest/windowsx86-64/debug/src/main/deploy")
-    } else {
-        into layout.buildDirectory.dir("test-results/frcUserProgramTest/linuxathena/debug/src/main/deploy")
-    } 
-}
+    // println prefix + "platform/version/" + suffix
+    // println OperatingSystem.current()
 
-tasks.register('copyDeployDirForUnitTestsRelease', Copy) {
-    from project.fileTree('src/main/deploy')
     if(OperatingSystem.current().isWindows()){
-        into layout.buildDirectory.dir("test-results/frcUserProgramTest/windowsx86-64/release/src/main/deploy")
+        // println "COPYING DEPLOY FOR WINDOWS"
+        // println "SRC: " + project.fileTree('src/main/deploy')
+        // println "TO"
+        // Copy debug
```

Comment:
Why all the commented out code?

Reply:
this is a bunch of debug statements for gradle things that I don't want to re-learn. Since we merge with squash I wanted to keep this here for now. We shouldn't do this in the 2023 code